src/dev/build/cli.ts

```diff
@@ -55,6 +55,9 @@ if (showHelp) {
         --skip-archives         {dim Don't produce tar/zip archives}
         --skip-os-packages      {dim Don't produce rpm/deb/docker packages}
         --all-platforms         {dim Produce archives for all platforms, not just this one}
+        --linux-x64             {dim Produce archives for only linux x64 platform}
```

Comment:
NIT: do you think we could make it `--linux` and have another flag for arch?

Reply:
good idea. I will do `--linux` , `--linux-arch` and `--darwin`.